src/bp-core/bp-core-catchuri.php

```diff
@@ -1522,7 +1522,8 @@ function bp_core_change_privacy_policy_link_on_private_network( $link, $privacy_
 
 		if ( $privacy_policy_url && $page_title ) {
 			$get_privacy_policy = get_post( $policy_page_id );
-			$get_content        = apply_filters( 'bp_privacy_policy_content', wp_kses_post( apply_filters( 'the_content', $get_privacy_policy->post_content ) ), $get_privacy_policy->post_content );
+			$get_content        = bp_strip_script_and_style_tags( apply_filters( 'the_content', $get_privacy_policy->post_content ) );
+			$get_content        = apply_filters( 'bp_privacy_policy_content', wp_kses_post( $get_content ), $get_privacy_policy->post_content );
```

Comment:
```suggestion
			$get_content        = apply_filters( 'bp_privacy_policy_content', wp_kses_post( bp_strip_script_and_style_tags( apply_filters( 'the_content', $get_privacy_policy->post_content ) ) ), $get_privacy_policy->post_content );
```
And remove 1525 line